analyses/copy_number_consensus_call/Snakefile

```diff
@@ -1,8 +1,6 @@
 ## Define the ending file(s) that we want
-OUTPUT= expand("../../scratch/interim/{sample}.{combined_caller}.{dupdel}.bed",
-               sample=config["samples"], 
-               combined_caller=["manta_cnvkit", "manta_freec", "cnvkit_freec"],
-               dupdel=["dup", "del"]),
+OUTPUT= expand("../../scratch/endpoints/{sample}.del_dup.merged.final.bed",
```

Comment:
If these are the final output files, we should probably put them in a results directory within the analysis directory.

Reply:
You are right, I forgot to put them in the "result" folder. I have made the changes to reflect this.